pkgs/tools/misc/tremor-rs/default.nix

```
@@ -3,22 +3,29 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "tremor";
-  version = "0.10.1";
+  version = "0.11.0";
 
   src = fetchFromGitHub {
     owner = "tremor-rs";
     repo = "tremor-runtime";
     rev = "v${version}";
-    sha256 = "1z1khxfdj2j0xf7dp0x2cd9kl6r4qicp7kc4p4sdky2yib66512y";
+    sha256 = "19g0ijkclrza6s0qcbwwh3lhlkisy00ffcl0c0d7dfqwrcisgz57";
   };
 
-  cargoSha256 = "sha256-rN/d6BL2d0D0ichQR6v0543Bh/Y2ktz8ExMH50M8B8c=";
+  cargoSha256 = "1xv205czb2z6qpqi6vslyrx2n21510qqa11i2hwya3jdcc9lkrsd";
 
   nativeBuildInputs = [ cmake pkg-config installShellFiles ];
 
   buildInputs = [ openssl ];
 
+  # TODO export TREMOR_PATH($out/lib) variable
   postInstall = ''
+    # Remove the static archive and the dynamic library
+    rm -rf $out/lib
```

```suggestion
    rm -r $out/lib
```

Do you know if we can disable building that all together? We do not need to compile something which we delete anyway.

---

I'm not sure. I'll ask on the Discord server and let you know.

---

It came from the `tremor-script` crate which was built because of the `--all` flag. I have changed it to `-p tremor-cli` so it doesn't build the static library anymore.